ocs_ci/framework/__init__.py

```diff
@@ -209,5 +209,32 @@ def switch_default_cluster_ctx(self):
         # because its a common arg
         self.switch_ctx(self.cluster_ctx.ENV_DATA["default_cluster_context_index"])
 
+    def get_provider_index(self):
+        for i, cluster in enumerate(self.clusters):
+            if cluster.ENV_DATA["cluster_type"] == "provider":
+                return i
```

Comment:
If there is no `provider`, this function will return `None` - I'm not sure, if it is expected or if it wouldn't better to raise some Exception, WDYT?

Reply:
Yes, sounds good.